stagger-haskell.cabal

```diff
@@ -17,6 +17,7 @@ library
                        containers,
                        deepseq,
                        errors               >= 1.4     && < 2.2,
+                       extra,
```

Comment:
Can you add a version to this? For libraries the convention is that deps bundled with GHC can be without a version, but other libraries should have version bounds. You can add `== 1.4.*`.